app/controllers/notifications_controller.rb

```diff
@@ -184,6 +184,12 @@ def sync
       format.html { redirect_back fallback_location: root_path }
       format.json { head :ok }
     end
+  rescue Octokit::BadGateway, Octokit::ServiceUnavailable
```

Comment:
What do you think about adding the real error to the server log?

Reply:
Losing it from the log is certainly silly, good catch. 👍